specs/Appearance.md

```diff
@@ -0,0 +1,72 @@
+Appearance (Theme) API
+===
+
+# Background
+This API's main use case is to set the overall appearance for WebView2. The options are similar to Edge: match the system default theme, light theme, or dark theme. 
+This API changes thing like dialogs, menus, and web content based on native flags. Please note this API will only set the overall appearance, but not theme.
+For reference, in the screenshot below this API is meant to expose the Overall Appearance section as a WebView2 API. 
+
+# Conceptual pages (How To)
+
+How to set theme in webview2. 
+1. Query interface. (Situational)
+2. Call put_Appearance to set the Appearance property. 
+3. The input parameter can be either COREWEBVIEW2_APPEARANCE_TYPE_SYSTEM, COREWEBVIEW2_APPEARANCE_TYPE_LIGHT, or COREWEBVIEW2_APPEARANCE_TYPE_DARK.
+
+
+# Examples
+
+### C++
+
+```cpp
+    wil::com_ptr<ICoreWebView2StagingController2> webViewStagingController2;
+    
+    bool ViewComponent::QuerySetAppearance(COREWEBVIEW2_APPEARANCE_TYPE appearance){
```

Comment:
This sample code should follow Microsoft coding style guidelines which have the  '{' on its own line and has 4 space indenting.
```suggestion
bool ViewComponent::QuerySetAppearance(COREWEBVIEW2_APPEARANCE_TYPE appearance)
{
```